.lintstagedrc.json

```
@@ -0,0 +1,7 @@
+{
+  "*.{js,jsx,ts,tsx,json}": [
+    "eslint src --ext .ts,.tsx",
+    "prettier --check \"src/**/*.{js,jsx,ts,tsx,json}\""
```

```suggestion
    "eslint",
    "prettier --check"
```
You don't want to specify files here, that would defeat the whole point of lint-staged. lint-staged will automatically pass the specific files that have been modified and need checking